_data/sites.json

```
@@ -19236,6 +19236,19 @@
         ]
     },
 
+    {
+        "name": "RemNote",
+        "url": "https://help.remnote.com/en/articles/8033036-deleting-your-account",
+        "difficulty": "easy",
+        "notes": "You can delete your account from Settings > Profile > Account Deletion > Delete Account",
+        "notes_es": "Para eliminar tu cuenta accede a Settings > Profile > Account Deletion > Delete Account",
```

We use the unicode right arrow:
```suggestion
        "notes": "You can delete your account from Settings ⮕ Profile ⮕ Account Deletion ⮕ Delete Account",
        "notes_es": "Para eliminar tu cuenta accede a Settings ⮕Profile ⮕ Account Deletion ⮕ Delete Account",
```